core-api/src/main/java/com/optimizely/ab/config/Experiment.java

```diff
@@ -16,17 +16,14 @@
  */
 package com.optimizely.ab.config;
 
-import com.fasterxml.jackson.annotation.JsonCreator;
-import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
-import com.fasterxml.jackson.annotation.JsonProperty;
-
-import java.util.Collections;
-import java.util.List;
-import java.util.Map;
+import com.fasterxml.jackson.annotation.*;
```

Comment:
Please update the license year to `2016-2018` for this and all the files you touched.